### PR TITLE
give syntax highlighting to readme snippets

### PR DESCRIPTION
**Pull request by Connoropolous**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ npm install --save-exact @holochain/conductor-api
...
-```
+```javascript
   const admin = await AdminWebsocket.connect(`http://localhost:8000`, TIMEOUT)
   await admin.generateAgentPubKey()
 ```
 
 ### Use AppWebsocket
-```
+```javascript
   const signalCb = (signal: AppSignal) => {
```
Comment (timotree3) [inline README.md]: This isn't javascript! :) I'm surprised it doesn't fail to parse
```suggestion
```typescript
  const signalCb = (signal: AppSignal) => {
```

File: README.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ npm install --save-exact @holochain/conductor-api
 ## Sample usage
 
 ### Use AdminWebsocket
-```
+```javascript
```
Comment (timotree3) [inline README.md]: Maybe future proof this too
```suggestion
```typescript
```

**Review by timotree3 — CHANGES_REQUESTED** [review]
I approve assuming we change it to `typescript`

**Comment by Connoropolous** [comment]
it doesn't appear to make any difference to the syntax highlighting, or truly recognize typescript, but it accepts it anyways I guess

**Review by timotree3 — APPROVED** [review]

**Pull request merged**